### DISCUSSION TITLE
InvalidCNAMEError on github

### DISCUSSION THREAD

**Discussion by AnatolyBuga** [community/community#146139] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I followed precisely [the steps](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site) to set up DNS on Squarespece to map it to github pages: 

![image](https://github.com/user-attachments/assets/7c07b5fe-542a-442c-9936-7ca490e05ed0)

However, I still get InvalidCNAMEError:
```
Both www.finestway.uk and its alternate name are improperly configured
Your site's DNS settings are using a custom subdomain, www.finestway.uk, that is not set up with a correct CNAME record. We recommend you change this to a CNAME record pointing to anatolybuga.github.io. For more information, see documentation (InvalidCNAMEError).
```
[Checking here](https://www.whatsmydns.net/#CNAME/www.finestway.uk) , my CNAME looks correct

Any idea why I might be getting this error?

**Comment by mecodeatlas** [comment]
Hey @AnatolyBuga!

This error is likely caused by inconsistencies in the domain configuration settings. I recommend reviewing this [troubleshooting guide](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages#cname-errors) specifically for CNAME errors. Additionally, [this Discussion](https://github.com/orgs/community/discussions/23479) addresses a very similar issue to yours and might provide helpful insights for resolving it :)
